### DISCUSSION TITLE
Why are deleted users kept as records?

### DISCUSSION THREAD

**Discussion by ghubz** [WeblateOrg/weblate#6002] [General]
Hi,

I've noticed that when deleting a user their profile remains, just email and full name is changed.

Considering that a user does not have any contributions whatsoever, why is this done and users are not fully deleted?

Thanks

**Comment by nijel** [comment]
All personal data is removed, but the contributions are still linked to that account. This still allows to differentiate between removed contributors in the history or suggestions.
